16_kata.py

```diff
@@ -0,0 +1,11 @@
+# https://www.codewars.com/kata/5552101f47fc5178b1000050/train/python
+
+def dig_pow(n, p):
+
+    list_variables = []
+    for nr in str(n):
```

Comment:
You mean here `nr` as shortcut for number? I think it's a Polish thing, because in english you would rather use `No.` abbreviation for a number. And here you actually take a digit from the number rather than number itself.